Automattic/wp-calypso
# Taxonomies: Add/Edit Categories from the Writing settings page

Use the TermFormDialog component to allow editing and adding categories right from the Settings writing page.

**testing instructions**
- Go to the /settings/writing/$site page
- Click "Add Category" and add new categories
- Click on a category to display the edition modal. You should be able to edit the term and the related posts should be updated correctly.

cc @timmyc
